ipa-core/src/bin/report_collector.rs

```diff
@@ -183,7 +183,7 @@ async fn ipa(
     let query_type: QueryType;
     match (security_model, &query_style) {
         (IpaSecurityModel::SemiHonest, IpaQueryStyle::Oprf) => {
-            query_type = QueryType::OprfIpa(ipa_query_config);
+            query_type = QueryType::OprfIpaRelaxedDpPadding(ipa_query_config);
```

Comment:
why not add an optional attribute in IpaQueryConfig called `relaxed_dp_padding`?

Reply:
I'm sure there are other ways of doing it, but this was the suggested approach from @andyleiserson.    I feel this way could be easy to extend in the future to supporting a RelaxedDP mode for DP noise generation in MPC, not just the relaxed for PaddingDP.

Reply:
I suppose the key question here is if semi-honest implies that relaxed DP is appropriate. the inverse is certainly true, i.e., malicious implies not relaxed DP, but there could be cases where we want semi-honest, but regular DP padding. 

if we're comfortable with tying these two together (and that probably makes sense for now), this works. otherwise, we may want this to be a distinct `IpaQueryStyle`. (which is incompatible with the malicious security model.)